code/SingleRecordAdmin.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace SilverStripe\Admin;
+
+use SilverStripe\Forms\Form;
+use SilverStripe\ORM\DataObject;
+
+/**
+ * The base class for admin sections that exist for editing a single record.
+ */
+abstract class SingleRecordAdmin extends LeftAndMain
+{
+    /**
+     * Determines if there should be a single record in the database that this admin edits.
+     *
+     * If this is not true, you need to provide a mechanism to tell the form which record it should be editing.
```

Comment:
```suggestion
     * If false, you need to provide a mechanism to tell the form, usually via getEditForm(), which record it should be editing.
```

Reply:
Will make the change, but won't add "usually via getEditForm()" because the only scenario we've actually implemented doesn't do it that way.